experimental/bionic-stemcell/openstack.yml

```diff
@@ -0,0 +1,6 @@
+- name: stemcell
+  path: /resource_pools/name=vms/stemcell?
+  type: replace
+  value:
+    sha1: e65cc7c5e4b40161ca63dc05228f38d60cd428a0
+    url: https://s3.amazonaws.com/bosh-core-stemcells-candidate/openstack/bosh-stemcell-168-openstack-kvm-ubuntu-bionic-go_agent-raw.tgz
```

Comment:
seems like this is a link to the raw stemcell, which has its own ops-file. We probably want the regular version in here? Or did you put the raw stemcell here intentionally?

Reply:
Not sure if the raw stemcell has its own ops file already, because I was not able to find any but created a PR for it (#403 )